### DISCUSSION TITLE
Welcome to Scala 3 Discussions!

### DISCUSSION THREAD

**Discussion by ckipp01** [scala/scala3#17927] [Announcements]
# Welcome to Scala 3 Discussions!

Thanks for stopping by. The discussions board here on the Scala 3 repository are meant to provide an easy way for those interested in Scala 3 to ask questions, discuss, and request new features. You'll find there are various categories meant to help triage your issue and be seen by the right people, so please do pay attention to the category you're submitting under.

## Creating Issues

If you're here creating an issue, you'll instead want to head over [here](https://github.com/scala/scala3/issues/new/choose). If you're not sure if your topic is an issue, discussion, or feature request, don't worry too much about it. We'll make sure it goes to the correct place.

## Contributing

If you're a contributor and looking for the contribution docs, please check out our [Contributors Guide](https://nightly.scala-lang.org/docs/contributing/index.html). If you see something missing, or can't find something in there, please do ask in here!

## Real-time chat

If you're looking for a more real-time chat experience, please head over to the official [Scala Discord Channel](https://discord.gg/scala).

_Note that this discussion board follows the [Scala Code of Conduct](https://scala-lang.org/conduct/)_.

**Comment by wrq** [comment]
This is a great idea. I can't imagine using Discord for something professional, what an enormous blunder that whole thing is.

I think sticking to FOSS platforms like Github or open protocols like IRC are definitely the way to go for software projects.
